core/src/main/java/org/apache/accumulo/core/clientImpl/TabletServerBatchWriter.java

```diff
@@ -210,8 +211,8 @@ public long getTimeOut() {
 
   public TabletServerBatchWriter(ClientContext context, BatchWriterConfig config) {
     this.context = context;
-    this.executor = context.threadPools()
-        .createGeneralScheduledExecutorService(this.context.getConfiguration());
+    this.executor = context.threadPools().createScheduledExecutorService(2,
```

Comment:
`createGeneralScheduledExecutorService` would use the `GENERAL_THREADPOOL_SIZE` property. 
This change switches to using a static number of two threads.

Do we want this thread pool size to be static?

Reply:
From what I could see, the BatchWriter only schedules two tasks.